eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -6,11 +6,15 @@ parameters:
   Recursive: $false
   CheckLinkGuidance: $true
   Urls: '(Get-ChildItem -Path ./ -Recurse -Include *.md)'
-  BranchReplaceRegex: "^(${env:SYSTEM_PULLREQUEST_SOURCEREPOSITORYURI}.*/(?:blob|tree)/)$(DefaultBranch)(/.*)$"
-  BranchReplacementName: "${env:SYSTEM_PULLREQUEST_SOURCECOMMITID}"
   Condition: succeeded() # If you want to run on failure for the link checker, set it to `Condition: succeededOrFailed()`.
 
 steps:
+  - pwsh: |
```

Comment:
I like this step but I think we should put it into its own template.

Reply:
Sure. I am considering to do the same as we have other places need to use the template.